src/ansys/additive/core/parametric_study/parametric_study.py

```diff
@@ -117,20 +117,21 @@ def run_simulations(
         type: list[SimulationType] | None = None,
         priority: int | None = None,
     ):
-        """Run the simulations in the parametric study with
-        ``SimulationStatus.PENDING`` in the ``ColumnNames.STATUS`` column.
-        Execution order is determined by the values in the
-        ``ColumnNames.PRIORITY`` column. Lower values are interpreted as having
+        """Run the simulations in the parametric study with ``Pending`` for
+        their ``Status`` value. Execution order is determined by their
+        ``Priority`` value. Lower values are interpreted as having
         higher priority and are run first.
 
         Parameters
         ----------
-        additive: :class:`Additive <ansys.additive.core.Additive>`
+        additive : Additive
             Additive service connection to use for running simulations.
         type : list[SimulationType], default: None
-            Type of simulations to run. If ``None``, all simulation types are run.
+            The type of simulations to run. If this value is ``None``,
```

Comment:
```suggestion
            Type of simulations to run. If this value is ``None``,
```
In the brief description, we drop the article so that the first word is an important one.